### PR TITLE
Add The QRL Foundation as member

### PR DESCRIPTION
**Pull request by cyyber**

### CODE REVIEW

File: assets/images/members/the-qrl-foundation/the-qrl-foundation.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="306px" height="165px" viewBox="0 0 306 165" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (vanbroup) [inline assets/images/members/the-qrl-foundation/the-qrl-foundation.svg:2]: Please make the logo auto scaling by removing the height and with:
```suggestion
<svg viewBox="0 0 306 165" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

**Review by vanbroup — CHANGES_REQUESTED** [review]

File: data/members/the-qrl-foundation.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+id: the-qrl-foundation
```
Comment (vanbroup) [inline data/members/the-qrl-foundation.yaml:1]: 'the' is probably not appropriate here, this is most likely also not the registered foundation name?
```suggestion
id: qrl-foundation
```

Comment (vanbroup) [reply data/members/the-qrl-foundation.yaml:1]: You will also need to rename the folder and logo as this id needs to be in sync.

Comment (cyyber) [reply data/members/the-qrl-foundation.yaml:1]: Foundation is registered in Switzerland with the name of Die QRL Stiftung which in English translates to The QRL Foundation. Die means The in English and Stiftung means the Foundation in English.

**Review by vanbroup — APPROVED** [review]

**Pull request merged**